.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
 script:
 - bundle exec rake db:create
 - bundle exec rake db:test:prepare
+- bundle exec rake db:seed RAILS_ENV=test
```

Comment:
Because there are associations and specs like be_valid checks that an instance of the associated object exist.